nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -0,0 +1,123 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.miniflux;
+
+  dbUser = "miniflux";
+  dbPassword = "miniflux";
+  dbHost = "localhost";
+  dbName = "miniflux";
+in
+
+{
+  options = {
+    services.miniflux = {
+      enable = mkEnableOption "miniflux";
+
+      user = mkOption {
+        type = types.str;
+        default = "miniflux";
+        description = ''
+          User account under which miniflux runs.
+        '';
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "miniflux";
+        description = ''
+          User group under which miniflux runs.
+        '';
+      };
+
+      listenAddress = mkOption {
+        type = types.str;
+        default = "localhost";
+        description = ''
+          The host the daemon will listen on.
+        '';
+      };
+
+      listenPort = mkOption  {
+        type = types.int;
+        default = 8080;
+        description = ''
+          The port the daemon will listen on.
+        '';
+      };
+
+      adminUser = mkOption  {
+        type = types.str;
+        default = "admin";
+        description = ''
+          Miniflux admin username.
+        '';
+      };
+
+      adminPassword = mkOption {
+        type = types.str;
+        default = "password";
+        description = ''
+          Miniflux admin password.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.postgresql.enable = true;
+
+    users = {
+      users.miniflux = {
+        name = cfg.user;
+        group = cfg.group;
+        description = "miniflux service user";
+        isSystemUser = true;
+        createHome = false;
+      };
+      groups.miniflux = {
+        name = cfg.group;
+      };
+    };
+
+    systemd.services.miniflux = {
+      description = "Miniflux service";
+      wantedBy = [ "multi-user.target" ];
+      requires = [ "postgresql.service" ];
+      after = [ "network.target" "postgresql.service" ];
+
+      serviceConfig = {
+        PermissionsStartOnly = true;
```

Comment:
PermissionsStartOnly is deprecated, please see https://github.com/NixOS/nixpkgs/issues/53852

Reply:
See my question at https://github.com/NixOS/nixpkgs/issues/53852#issuecomment-471009451

Reply:
I'll leave it that way because we need https://github.com/NixOS/nixpkgs/pull/56720 to be merged to replace `PermissionsStartOnly` functionality.

Reply:
There are several ways to achieve this without the PR I mentioned. I will try to get the postgresql PR through as quick as I can, though.